lesson03/assignment/js/temp

```diff
@@ -0,0 +1,14 @@
+<section class="profile">
+  <img src="./img/unsplash-headshot.jpg" alt="profile-picture">
+  <h1>${person.name}</h1>
+  <p>${person.jobTitle}</p>
+</section>
+<section class="info">
+  <p><b>Company:</b>${person.company}</p>
+  <p><b>Experience:</b>${person.experience}</p>
+  <p><b>School:</b>${person.school}</p>
+  <p><b>Major:</b>${person.major}</p>
+  <p><b>Email:</b>${person.email}</p>
+  <div><img src="./img/linkedin.svg" alt=""><a href="">${person.linkedInUrl}</a></div>
+</section>
+</article>
```

Comment:
This looks like a practice file, building out the HTML you wanted to create in the JS function, right? If you want to commit this, for future reference, keeping it as a file with no extension is a good idea. I'd also add a comment to the top explaining its purpose.